src/components/navigation/navbarIcons.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+export const userIcon = (
```

Comment:
There might be an easier way to import the icons, but if not this is fine

Reply:
I'm now storing the SVGs as individual files, so the imports/exports should be closer to normal. However, there are a lot more import statements in navbar.jsx now. Would you happen to know away to compress these imports back into one line?

Reply:
I guess my original comment for this was bad, I meant like I was wondering if there was an icon library we could use so that we don't have to store each icon. Ive used this one before:  https://react-icons.github.io/react-icons/. I'll run this by Mandy and see what she thinks. about these icons as opposed to hers.

Reply:
I ran it by Mandy and she gave us the go ahead to use the above font library. You'll need to install the package, and then just find and use similar icons to the ones you are currently using.

Reply:
Thanks, that package was very helpful! I swapped out the icons so the imports are much cleaner now.